python/seldon_core/wrapper.py

```diff
@@ -169,19 +169,40 @@ def Metrics():
 def _set_flask_app_configs(app):
     """
     Set the configs for the flask app based on environment variables
+    See https://flask.palletsprojects.com/config/#builtin-configuration-values
     :param app:
     :return:
     """
-    env_to_config_map = {
-        "FLASK_JSONIFY_PRETTYPRINT_REGULAR": "JSONIFY_PRETTYPRINT_REGULAR",
-        "FLASK_JSON_SORT_KEYS": "JSON_SORT_KEYS",
-    }
-
-    for env_var, config_name in env_to_config_map.items():
-        if os.environ.get(env_var):
-            # Environment variables come as strings, convert them to boolean
-            bool_env_value = os.environ.get(env_var).lower() == "true"
-            app.config[config_name] = bool_env_value
+    FLASK_CONFIG_IDENTIFIER = "FLASK_"
+    FLASK_CONFIGS_BOOL = [
+        "DEBUG",
+        "EXPLAIN_TEMPLATE_LOADING",
+        "JSONIFY_PRETTYPRINT_REGULAR",
+        "JSON_SORT_KEYS",
+        "PROPAGATE_EXCEPTIONS",
+        "PRESERVE_CONTEXT_ON_EXCEPTION",
+        "SESSION_COOKIE_HTTPONLY",
+        "SESSION_COOKIE_SECURE",
+        "SESSION_REFRESH_EACH_REQUEST",
+        "TEMPLATES_AUTO_RELOAD",
+        "TESTING",
+        "TRAP_HTTP_EXCEPTIONS",
+        "TRAP_BAD_REQUEST_ERRORS",
+        "USE_X_SENDFILE",
+    ]
+
+    for env_var, value in os.environ.items():
+        if not env_var.startswith(FLASK_CONFIG_IDENTIFIER):
+            continue
+        flask_config = env_var.replace(FLASK_CONFIG_IDENTIFIER, "")
+        if flask_config not in FLASK_CONFIGS_BOOL:
+            logger.warning(
+                f"The Flask Configuration Parameter, {flask_config}, is not configured for use with Seldon"
+            )
+            continue
+        # Environment variables come as strings, convert them to boolean
+        app.config[flask_config] = value.lower() == "true"
```

Comment:
There is a `getenv_as_bool` utility method in `seldon_core/utils.py` which could be useful here. Not that it's required to use it though! Just flagging it in case you find it useful.

Reply:
Put it in.